include/rapidjson/rapidjson.h

```diff
@@ -169,9 +169,13 @@
 #include "msinttypes/inttypes.h"
 #else
 // Other compilers should have this.
+// Borland's Clang derived compiler have many of these declared in stdlib.h and including
+// these causes compilation errors
```

Comment:
Should't we then include `<stdlib.h>` for `__BORLANDC__` instead?

Reply:
stdlib.h is already included (or included later, thus the compilation bug with abs()). This compiled successfully upon testing.

Reply:
Including headers of standard library should not generate error.
If this is a compiler bug, isn't that should be fixed by the compiler?